Documentation/FrameworkOverview.md

```diff
@@ -8,36 +8,33 @@ learning about new modules and finding more specific documentation.
 For examples and help understanding how to use RAC, see the [README][] or
 the [Design Guidelines][].
 
-## Streams
+## Signals
+
+A **signal**, represented by the [Signal][] class, is any series of objects.
 
-A **stream**, represented by the [RACStream][] abstract class, is any series of
-object values.
+<!-- TODO: This is something I don't know, is it still true for Signal?  -->
+<!-- Signals are [monads][]. Among other things, this allows complex operations to be
+built on a few basic primitives (`-bind:` in particular). [Signal][] also
+implements the equivalent of the [Monoid][] and [MonadZip][] typeclasses from
+[Haskell][]. -->
 
 Values may be available immediately or in the future, but must be retrieved
 sequentially. There is no way to retrieve the second value of a stream without
 evaluating or waiting for the first value.
```

Comment:
True. Doesn't this mean that it was not even true (for all signals) before? As I understand it, your comment also applies to hot signals. 

Could it be that this paragraph originally tried to say something along the lines of _"theres no random access to the values of a stream"_ / _"an observer can only evaluate the events in the order they are sent over the signal [as long as he is observing]"_?